src/styles/components/_NotFoundButton.scss

```diff
@@ -0,0 +1,48 @@
+.notfound-404 {
+    position: relative;
+    height: 200px;
+    h2 {
+      font-family: 'Montserrat', sans-serif;
+      font-size: 22px;
+      font-weight: 700;
+      text-transform: uppercase;
+      color: #ffffff;
+      margin-top: 20px;
+      margin-bottom: 15px;
+    }
+  }
+
+  .home-btn {
+    font-family: 'Montserrat', sans-serif;
+    display: inline-block;
+    font-weight: 700;
+    text-decoration: none;
+    background-color: transparent;
+    border: 2px solid transparent;
+    text-transform: uppercase;
+    padding: 13px 25px;
+    font-size: 18px;
+    border-radius: 40px;
```

Comment:
This too. The buttons on our designs are not with radius. I think there should be uniformity in all our buttons. You can consider creating the button component in a `common folder`  so that everybody can use it. The designs will be a perfect guide for the look of the button.

Reply:
If I get you right, you mean the style of the button should be passed as props?

Reply:
Not exactly. Create the button component and pass it to the 404 page layout to render